### PR TITLE
fix oid4vci tests

### LINKED ISSUE #30079: The OID4VC tests break automation
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

account/api

### Describe the bug

The [OID4VC UI commit](https://github.com/keycloak/keycloak/pull/29174) introduced "test" which never run anything since it was using property access instead of function calls: Note the missing `();` denoting function calls in [the test](https://github.com/keycloak/keycloak/pull/29174/files#diff-bcc8430b113f420f11c5ec5071ad5c511f8aa632a712c28da62c13c5a2a5a5b3.)

Now the account tests passed because they were never run, see #29976. Once this is fixed, the account tests fail due to the test issues, see https://github.com/keycloak/keycloak/actions/runs/9348103213/job/25726758123?pr=29973#step:8:116.

### Version

main

### Regression

- [X] The issue is a regression

### Expected behavior

The test tests desired behaviour and passes.

### Actual behavior

The test does not test anything, but fails regardless due to missing `oid4vci` control.

This is the place where it fails:

image

### How to Reproduce?

Run account e2e tests.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by wistefan**
fixes #30079

### CODE REVIEW

File: js/apps/account-ui/test/oid4vci/oid4vci.spec.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    await login(page, "test-user", "test", realm);
+
+    await expect(page.getByTestId("qr-code")).toBeHidden();
+
+    await page.getByTestId("oid4vci").click();
+    await page.getByTestId("credential-select").click();
+
+    await expect(
+      page.getByTestId("select-verifiable-credential"),
+    ).toBeVisible(),
```
Comment (jonkoops) [inline js/apps/account-ui/test/oid4vci/oid4vci.spec.ts:17]: Looks like some code formatting issues here, make sure you have the linter enabled. If you are using VSCode I can recommend the [ESLint plugin](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint). There are also options for [JetBrains stuff](https://www.jetbrains.com/help/idea/eslint.html)

Comment (wistefan) [reply js/apps/account-ui/test/oid4vci/oid4vci.spec.ts:17]: Thats what the linter in the pipeline enforced(e.g. pnpm --fail-if-no-match --filter @keycloak/keycloak-admin-client lint). Looks weird to me to,  but I thought its on intention:)

Comment (jonkoops) [reply js/apps/account-ui/test/oid4vci/oid4vci.spec.ts:17]: The comma should be a semicolon here.

Comment (wistefan) [reply js/apps/account-ui/test/oid4vci/oid4vci.spec.ts]: Fixed in [14a182e](https://github.com/keycloak/keycloak/pull/30127/commits/14a182ede2f779656143ce3eb229efb3af47b5ba)

**Review by hmlnarik — DISMISSED** [review]
The Account UI E2E tests fail here. Please fix

**Comment by ahus1** [comment]
@edewit / @jonkoops / @hmlnarik - the tests are now green. Could one of you please re-review to make this part of KC25?

**Review by jonkoops — APPROVED** [review]
LGTM

**Pull request merged**